posts/2022-01-18-beta-release-post.adoc

```diff
@@ -0,0 +1,146 @@
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<ssl, Support new change in the SSL certificate with urlencoded string>>
+
+[#ssl]
+=== Support new change in the SSL certificate with urlencoded string
+
+An `X.509` certificate is a digital certificate which is used to manage identity and security in internet communications and computer networking.
+
+The `X.509` certificate can be sent by the intermediate servers (like web server, proxy server or load balancer) to an application server. Due to the deprecation of long lines folding in the `HTTP` header continuation, some of the open source intermediate servers start sending a client certificate in the Privacy-Enhanced Mail (`PEM`) url-encoded certificate to application server.
+
+Currently, WebSphere Application Server and Liberty Server do not support this new url-encoded certificate. Any request with this new format is rejected and the request is terminated.
+
+An example of a truncated url-encoded certificate sent to application server:app-name:
+
+[source]
+----
+[-----BEGIN%20CERTIFICATE-----%0AMIIDODCCAiCgAwIBAgIIHbO76YfCH1cwDQYJKoZIhvcNAQELBQAwOjELMAkGA1UE%0ABhMCVVMxDDAKBgNVBAoTA0lCTTEMMAoGA1UECxMDVFdTMQ8wDQYDVQQDEwZDbGll%0AbnQwHhcNMTUxMTAzMTYyODM1WhcNMzUxMDMwMTYyODM1WjA6MQswCQYDVQQGEwJV%0AUzEMMAoGA1UEChMDSUJNMQwwCgYDVQQLEwNUV1MxDz%2F%2Bpo%2FF%2Bzi%2F0sfAUwRdfgMm%2FAKAGzwMQufOYeKCgMULtq14QAJQnLmq4M%2FM00%0AC5QyeYtdaTou%2BMsLmoa1tkq2VSDVxAcktJyRSRsox36G7EHDLV4U2gtR6xczNEjw%0A2%2Bsj772FjdAMXRSR%0A-----END%20CERTIFICATE-----%0A]
+----
+
+Note: 
+* (1) The header (i.e BEGIN ) and footer (i.e END) information are appended before sending to application server.
+* (2) The encoded character (i.e %) presents in the certificate.
+
+The IBM application server code is updated to auto detect and process this new format.
+
+For more information, check out the link:https://github.com/OpenLiberty/open-liberty/issues/11680[GitHub issue] for this new feature.
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
```

Comment:
```suggestion
To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 17, Java SE 11, or Java SE 8.
```